_episodes/l1-06-releases_tags.md

```diff
@@ -3,19 +3,355 @@ title: "Code versions, releases and tags"
 teaching: 0 # TBD
 exercises: 0 # TBD
 questions:
+- "What is a Git tag and how does it differ from a branch?"
 - "How and when should I release a new version of my code?"
 - "What is the difference between major and minor version changes?"
-- "How can I effectively communicate what has changed between code version?"
-
+- "How can I effectively communicate what has changed between versions?"
+- "How can I tag commits?"
```

Comment:
Maybe move this up, to keep a more logical order?

Reply:
Good point.